activesupport/lib/active_support/hash_with_indifferent_access.rb

```diff
@@ -393,6 +393,19 @@ def set_defaults(target) # :doc:
           target.default = default
         end
       end
+
+      def update_with_single_argument(other_hash, block)
```

Comment:
do we want a `# :doc:` here as for the other private methods?

Reply:
No, we don't.